### PR TITLE
Update search indexer

### LINKED ISSUE #69: Update `SearchIndexer` for the new API docs format
[This method](https://github.com/ember-learn/ember-cli-addon-docs/blob/versioned-deploys/lib/broccoli/search-indexer.js#L81-L104) and the [logic that decides to invoke it](https://github.com/ember-learn/ember-cli-addon-docs/blob/versioned-deploys/lib/broccoli/search-indexer.js#L48-L49) need to be updated to consume the new documentation structure.

### PR DESCRIPTION
**Pull request by samselikoff**
- fixes #69

### CODE REVIEW

**Comment by samselikoff** [comment]
I want to update the search UI but figured we could land this first

File: addon/components/docs-header/search-box/component.js [hunk 1]
```diff
@@ -13,8 +20,18 @@ export default Component.extend(EKMixin, {
...
 
     this.set('keyboardActivated', true);
+    this.get('fetchProject').perform();
   },
 
+  // TODO: The searchbox doesn't work without the project being fetched.
+  // We should move this logic (and everywhere else in the code that's fetching
+  // the project) within a new addonDocs service that wires all that up together.
+  // I think it's fine if our Docs-* components assume there is a single global
+  // project.
```
Comment (dfreeman) [inline addon/components/docs-header/search-box/component.js]: 👍

File: tests/acceptance/search-test.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    await visit('/');
+    await appPage.fillInSearchQuery('quickstart');
+
+    await waitUntil(() => appPage.searchResults.items.length > 0);
+
+    await click(find('[data-test-search-result] a'));
+    assert.equal(currentURL(), '/docs/quickstart');
+  });
+
+  test('search works for API pages', async function(assert) {
```
Comment (dfreeman) [inline tests/acceptance/search-test.js]: ❤️

**Comment by samselikoff** [comment]
woot, ty for review!

**Pull request merged**